tests/pytests/unit/modules/test_yumpkg.py

```diff
@@ -1747,3 +1921,36 @@ def test_call_yum_with_kwargs():
                 python_shell=True,
                 username="Darth Vader",
             )
+
+
+@pytest.mark.skipif(not salt.utils.systemd.booted(), reason="Requires systemd")
+@patch("salt.modules.yumpkg._yum", Mock(return_value="dnf"))
```

Comment:
`with patch(....):` instead of the decorator?

https://engblog.nextdoor.com/what-your-mocks-do-when-you-aren-t-looking-b278e0d9e201